app/basicComponents/SmInput.js

```diff
@@ -28,11 +28,26 @@ const Input = styled.input`
   outline: none;
 `;
 
+const ErrorsSection = styled.div`
+  position: relative;
+  height: 0;
+`;
+
+const ErrorMessageWrapper = styled.div`
```

Comment:
don't use position: absolute  - it's prone to UI issues of cut text and/or overflowing on other ui elements

Reply:
I've wrapped the Input and error sections to solve this. Otherwise the error section position is dependent upon the outside flex styling.